### ISSUE TITLE
Closed flags should only be visible to supervisors

### ISSUE THREAD

**Issue by gjacovella** [NRGI/rgi-assessment-tool#524]
My peer reviewer for Cameroon was able to read the closed flags that were addressed to the researcher. He got confused and thought that the flags were for him. Can we make sure that both peer reviewers and researchers are not able to access flags once they're closed, while keeping all flags accessible to supervisors? 

The researchers and peer reviewers should only be able to see open flags and flags which have been marked as unresolved. @alexander-elgin @ahasemann

**Comment by alexander-elgin** [comment]
Flag addressing was shown incorrectly. It was shown that all flags are addressed to the current user (independently of the actual `addressed` value). I've fixed it. At the moment users to whom the flag was not addressed see the following (as it was implemented originally):
![screenshot-localhost 3030 2016-08-24 14-02-20](https://cloud.githubusercontent.com/assets/1811689/17923139/746d0612-6a04-11e6-8a74-5fa8625b6d86.png)
Is it still required to hide the flags from the users whom the flags were not addressed?
